core/src/main/scala/io/qbeast/core/transform/LinearTransformation.scala

```diff
@@ -77,7 +77,7 @@ case class LinearTransformation(
           otherNullValue,
           orderedDataType)
           .asInstanceOf[Transformation]
-      case IdentityTransformation(newVal) =>
+      case IdentityToZeroTransformation(newVal) =>
         val otherNullValue =
           LinearTransformationUtils.generateRandomNumber(
             min(minNumber, newVal),
```

Comment:
I understand the name change comes from the fact that the IdentityTransformation will always map the values to zero for the index by default, right?

Reply:
Yes, it's purely a naming convention :smile: